.github/workflows/build.yml

```diff
@@ -107,6 +102,13 @@ jobs:
         with:
           string: ${{ env.IMAGE_REGISTRY }}
 
+      - name: Build Image
+        id: build-image
+        shell: bash
+        run: |
+          IMAGE_FULL="${{ steps.registry_case.outputs.lowercase }}/${IMAGE_NAME}"
+          just build "${IMAGE_FULL}" "${DEFAULT_TAG}"
```

Comment:
We need to refactor the build-vm and run-vm ones after this gets merged. This will kind of break it I think..?

Reply:
That's the main reason this is still wip, but wanted to get this pushed before I called it a night

Reply:
Worst case dont worry we can work on it outside of this PR. I want to keep the achillobator `just run-vm 1 raw` workflow, it is super cleeeean!

Reply:
build-vm and run-vm recipes are working now. I did refactor run-vm to run in a container (blatantly stolen from the bluefin run-iso job)